src/components/List/TestDataUserList.ts

```diff
@@ -0,0 +1,53 @@
+export interface UserType {
+  name: string;
+  role?: string;
+  users?: UserType[];
+}
+
+export const testUsers: UserType[] = [
+  {
+    name: 'Dan Daglig Leder',
```

Comment:
We should probably add an id field to each user, which we can use as unique identifier in our code 😊